src/App.css

```diff
@@ -0,0 +1,72 @@
+:root {
+  --gray-color: #616161;
+  --gray-light: #a1a1a1;
+  --black-color: hsl(220, 15%, 12%);
+  --yellow-light: #f7e7a1;
+  --primary-notes: #0f56b3;
+}
+
+.App {
+  font-family: Poppins;
+  min-width: 100vw;
+  min-height: 100vh;
+
```

Comment:
remove the extra space

Reply:
Thanks for pointing it out, I have made the changes in the latest commits.